mission_control/navigator_missions/nav_missions/scan_the_code.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+import txros
+import tf
+import numpy as np
+import navigator_tools
+import math as math
+import matplotlib.pyplot as plt
+import rospy
+import navigator_msgs.srv as navigator_srvs
+
+
+@txros.util.cancellableInlineCallbacks
+def main(navigator):
+    navigator.change_wrench("autonomous")
+    yield navigator.vision_request("scan_the_code_activate")
+
+    # activate stereo model fitter
+    granularity = 6
+    distance = 3
+    angle = 360/granularity * 3.14/180
+    n = [0,0]
+    # this is the absolute position, give it the relative position option
+    #base = [-5,5,0]
+    base = navigator.move.latlon() + navigator.move.position
+
+    for i in range(0,granularity+1):
+        n[0] = distance * math.cos(i*angle) ; n[1] = distance * math.sin(i*angle)
```

Comment:
This could be cleaned up, generally in python if you're doing basic math on arrays, numpy will do it all at once so `n = distance * np.array([math.cos, math.sin)]` looks a lot cleaner then the individual assignment imo